src/reports/components/instance-details.scss

```diff
@@ -14,12 +14,36 @@
 
 .instance-details-card {
     border-radius: 4px;
-    box-shadow: 0px 0.6px 1.8px rgba(0, 0, 0, 0.108), 0px 3.2px 7.2px rgba(0, 0, 0, 0.132);
+    outline-style: 'border-style';
+    box-shadow: 0px 0.6px 1.8px $box-shadow-108, 0px 3.2px 7.2px $box-shadow-132;
     margin-left: 24px;
     margin-bottom: 16px;
+    cursor: pointer;
     @include fill-available-width;
 }
 
+.instance-details-card.selected {
+    border: 5px solid $communication-tint-10;
+}
+
+.instance-details-card:focus {
+    outline: 2px solid $primary-text;
```

Comment:
Per the Figma, there should be a 2px gap between the focus ring and the card. To get this, you might be able to use `outline-offset: 2px;` on these focus styles.

Reply:
Is it? I couldn't figure out that there was a gap between the two borders, will add it. Thanks for catching this.

Reply:
Updated